radio/util/hw_defs/pot_config.py

```diff
@@ -15,15 +15,15 @@
         "P2": {"default": "POT_CENTER"}
     },
     "pl18": {
-        "P1": {"default": "POT_CENTER"},
-        "P2": {"default": "POT_CENTER"},
-        "P3": {"default": "POT_CENTER"},
+        "P1": {"default": "POT"},
+        "P2": {"default": "POT"},
+        "P3": {"default": "POT"},
         "SL1": {"default": "SLIDER"},
         "SL2": {"default": "SLIDER"}
     },
     "pl18ev": {
         "P1": {"default": "POT_CENTER"},
-        "P2": {"default": "POT_CENTER"},
+        "P2": {"default": "POT"},
```

Comment:
Mistake? Pots on the EV are all clicky pots with a centre dot mark

Reply:
And my PL18 all 3 pots do not have center mark

Reply:
Interesting... my PL18EV looks exactly like the stock photo on the Flysky website 🤪 

https://www.flysky-cn.com/paladin-evdescription

![image](https://github.com/EdgeTX/edgetx/assets/5500713/29ec02c8-d092-4acc-a297-60153c1efa89)

Reply:
PL18 and PL18EV have different VRA/VRB/VRC config. PL18EV has noticeable center detent on VRA and VRC, whereas VRB has ratcheting over the whole range and (at least I) cannot blindly detect the center point.

Reply:
Weird... looks like there are some variations even within the PL18EV... my PL18EV all three VR are the same (what you refer to as ratcheting with no perceptible centre detent)... and I can see at least one version of the PL18EV with only two VR pots... https://youtu.be/5C5pOiyowng?si=Vbe-0niXBIL57p5Z&t=9

Reply:
The radio with two pots is Paladin EV Lite: https://www.flysky-cn.com/paladinevlitedescription